File: berufsschule/11/AP/CPP/Hausaufgaben/3_OnlineBuch/OnlineBuch.cpp
#include "OnlineBuch.h"
#include "Buch.h"
#include "Kapitel.h"

OnlineBuch::OnlineBuch()
{
	anzahlAutoren = 0;
	anzahlKapitel = 0;
}
OnlineBuch::~OnlineBuch()
{
}

int OnlineBuch::getAnzahlAutoren()
{
	return anzahlAutoren;
}
void OnlineBuch::setAnzahlAutoren(int zahl)
{
	anzahlAutoren = zahl;
}
int OnlineBuch::getAnzahlKapitel()
{
	return anzahlKapitel;
}
void OnlineBuch::setAnzahlKapitel(int zahl)
{
	anzahlKapitel = zahl;
}

File: berufsschule/11/AP/CPP/Hausaufgaben/3_OnlineBuch_Java/src/OnlineBuch.java

public class OnlineBuch extends Buch
{
	private int anzahlAutoren;
	private String autoren[] = new String[anzahlAutoren];
	private int anzahlKapitel;
	private Kapitel kapitel[] = new Kapitel[anzahlKapitel];
	
	
	public int getAnzahlAutoren() 
	{
		return anzahlAutoren;
	}
	public void setAnzahlAutoren(int anzahlAutoren)
	{
		this.anzahlAutoren = anzahlAutoren;
	}
	public String[] getAutoren()
	{
		return autoren;
	}
	public void setAutoren(String[] autoren)
	{
		this.autoren = autoren;
	}
	public int getAnzahlKapitel() 
	{
		return anzahlKapitel;
	}
	public void setAnzahlKapitel(int anzahlKapitel)
	{
		this.anzahlKapitel = anzahlKapitel;
	}
	public Kapitel[] getKapitel() 
	{
		return kapitel;
	}
	public void setKapitel(Kapitel[] kapitel) 
	{
		this.kapitel = kapitel;
	}	
}
